utils/b2d.go

```diff
@@ -94,6 +94,14 @@ func (b *B2dUtils) GetLatestBoot2DockerReleaseURL() (string, error) {
 	return isoUrl, nil
 }
 
+func removeFileIfExists(string name) {
+	if err := os.Stat(name); err == nil {
+		if err := os.Remove(name); err != nil {
```

Comment:
This is a race...  Why not use [`os.IsNotExist`](https://golang.org/pkg/os/#IsNotExist)?